testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -54,6 +54,7 @@ public abstract class LayersTestBase {
             "org.jboss.as.xts",
             // TODO should an undertow layer specify this?
             "org.wildfly.event.logger",
+            "org.wildfly.security.elytron-dynamic-ssl",
```

Comment:
In my opinion, this is not the expected fix.

`test-all-layers` installation contains the `elytron` galleon layer, so the `org.wildfly.security.elytron-dynamic-ssl` should have been provisioned by the `test-all-layers`, which makes this change unnecessary.

The fix should be in WildFly Core PR, I'll comment more about it there. The problem is that `elytron` galleon layer is not provisioning `org.wildfly.security.elytron-dynamic-ssl` package.

Reply:
@bstansberry I guess we also are going to need a solution to tweak all of these galleon layers tests based on provisioning stability levels. Not sure if we have already discussed the approach for them.